src/Player/PlayerState/PlayerStateGroundSpin.cpp

```diff
@@ -0,0 +1,63 @@
+#include "Player/PlayerState/PlayerStateGroundSpin.h"
+#include "Player/PlayerActionGroundMoveControl.h"
+#include "Player/PlayerInput.h"
+#include "Player/PlayerAnimator.h"
+#include "Player/PlayerConst.h"
+#include "Util/Math.h"
+#include <al/Library/Nerve/NerveSetupUtil.h>
+#include <al/Library/Nerve/NerveUtil.h>
+#include <al/Library/Math/MathUtil.h>
+#include <al/Library/LiveActor/ActorMovementFunction.h>
+#include <al/Library/LiveActor/ActorPoseKeeper.h>
+
+namespace {
+    NERVE_IMPL(PlayerStateGroundSpin, GroundSpin);
+    struct {
+        NERVE_MAKE(PlayerStateGroundSpin, GroundSpin);
+    } nrvPlayerStateGroundSpin;
+}
+
+PlayerStateGroundSpin::PlayerStateGroundSpin(al::LiveActor *parent, const IUsePlayerCollision *collision,
+                                             const PlayerInput *input, const PlayerConst *playerConst,
+                                             PlayerAnimator *animator) : ActorStateBase("地上スピン", parent),
+                                                                         mCollision(collision), mPlayerInput(input), mPlayerConst(playerConst),
+                                                                         mPlayerAnimator(animator) {
+    mIsSpinClockwise = false;
```

Comment:
Is this line required for matching, or does it also match without it due to it being initialized in the header as well?